src/test/functional/features/homepage.js

```diff
@@ -78,11 +78,20 @@ Scenario('ET gender details page', async () => {
   I.see('Sex and gender identity');
 });
 
-async function etpageFlow() {
-  I.amOnPage(testConfig.testUrl);
-  I.waitForText('Make a claim to an employment tribunal', waitSeconds);
-  I.waitForText('Start now', waitSeconds);
+Scenario('ET homepage verify cached item configuration on a new tab', () => {
+  etpageFlow();
+  I.checkOption('input[id=lip-or-representative]');
+  I.openNewTab();
+  I.amOnPage('/lip-or-representative');
+  I.seeInCurrentUrl('/lip-or-representative');
+  I.see('I’m representing myself and making my own claim');
+  I.dontSeeCheckboxIsChecked('representingMyself');
+}).tag('@RET-1014');
+
+function etpageFlow() {
+  I.amOnPage('/');
```

Comment:
url vary from environment to environment. can we keep testUrl only?

Reply:
it should pick up what you have in config file.

Reply:
cool, approved